odl/space/ntuples.py

```diff
@@ -1320,7 +1349,7 @@ def adjoint(self):
                                       ''.format(self.domain.field,
                                                 self.range.field))
         return MatVecOperator(self.range, self.domain,
-                              np.asarray(np.asmatrix(self.matrix).H))
+                              self.matrix.conj().T)
```

Comment:
But it represents a matrix. Or do you mean the implication that this might be a `numpy.matrix`?

A bit of `matrix` would still be nice in the name, what about `mat_array`?